test/unit/controllers/DevelopersControllerSpec.scala

```diff
@@ -91,23 +94,23 @@ class DevelopersControllerSpec extends UnitSpec with MockitoSugar with WithFakeA
     "developersPage" should {
 
       "default to page 1 with 100 items in table" in new Setup {
-        givenASuccessfulLogin
-        givenNoDataSuppliedDelegateServices
-        val result = await(developersController.developersPage(None, None)(aLoggedInRequest))
+        givenASuccessfulLogin()
```

Comment:
You don't need the parentheses on the 'givens' (several places in this file). Also remove the type annotations (e.g. `Result`, `Seq[Developer]`) throughout